frontend/src/utils.js

```diff
@@ -69,9 +69,35 @@ function gcmCompliant() {
   }
 }
 
+function findMetric(href) {
+  switch(href) {
+    case 'https://www.mozilla.org/':
+      return 'mozilla';
+    case 'https://www.mozilla.org/about/legal':
+      return 'legal';
+    case 'https://testpilot.firefox.com/about':
+      return 'about';
+    case 'https://testpilot.firefox.com/privacy':
+      return 'privacy';
+    case 'https://testpilot.firefox.com/terms':
+      return 'terms';
```

Comment:
They're intended to be the footer links. If there are additional link destinations I missed, feel free to disambiguate or add to the list as appropriate as long as you update the docs; I pieced this together from the mocks.